xclim/sdba/nbutils.py

```diff
@@ -14,25 +14,14 @@
 USE_NANQUANTILE = environ.get("USE_NANQUANTILE", False)
 
 
-@njit
-def _quantile(arr, q):
-    if arr.ndim == 1:
-        out = np.empty((q.size,), dtype=arr.dtype)
-        out[:] = _choosequantile(arr, q)
-    else:
-        out = np.empty((arr.shape[0], q.size), dtype=arr.dtype)
-        for index in range(out.shape[0]):
-            out[index] = _choosequantile(arr[index], q)
-    return out
-
-
 @njit(
     [
         int64(float32[:]),
         int64(float64[:]),
     ],
     fastmath=False,
     nogil=True,
+    cache=False,
```

Comment:
This was removed because it slows down the import of  `xclim` a lot (and by proxy, `xscen`). Many users will never use `sdba`, so it made sense to add the compilation burden only to those who did. And even then, waiting for the import seemed much more frustrating than waiting as few seconds more for your complex and heavy computation. See https://github.com/Ouranosinc/xclim/pull/1378.

Reply:
An option would be to use [numba.cc](https://numba.readthedocs.io/en/stable/user/pycc.html) for ahead-of-time compilation during module install.

Reply:
Oh I think I misunderstood, I thought you were talking about pre-compilation in general. Caching is not a problem, I just turned them off to make sure that my implementations for pre-compilation were working.

Reply:
Ah! All good then, as long as we re-activate it before merging.

Reply:
> Todo: turn on cache=True

seems to contradict the discussion above. We leave caching turned off?